image_manager_main.cpp

```diff
@@ -22,6 +23,14 @@ int main()
 
     try
     {
+        if (std::filesystem::is_directory(IMG_UPLOAD_DIR))
```

Comment:
This logic would execute even on non-mmc systems. Instead of adding an ifdef block for MMC configuration, let's add the remove in the systemd file, in that way the changes are all contained in the new mmc service, we can use rm -rf